### ISSUE TITLE
Profile: open Media tab by default

### ISSUE THREAD

**Issue by phoe** [tootsuite/mastodon#10127]
### Pitch

Currently, on an open Mastodon profile, the three available tabs are Toots, Toots and replies, and Media. Toots is the one open by default.

The idea is to make it possible for the user to choose which tab they would like to display by default on their profile.

### Motivation

The benefit is that artists might want to showcase their creations by default, rather than their toots and boosts. It would give their works more visibility by default.

People need to be aware that they need to click on the Media tab in order to see that artist's posts. It is not immediately obvious.

The mechanism of toot-pinning is a workaround, but is manual.

**Comment by MaiaDereva** [comment]
I agree! Adding a few details when you hover over the image with the mouse (number of likes, comments and boosts) would round off this basic portfolio feature.
